### ISSUE TITLE
enable_object_processing

### ISSUE THREAD

**Issue by jtenniswood** [xblax/flashforge_ad5m_klipper_mod#242]
Could we added 'enable_object_processing: True' to the file manager setting in moon raker.conf to allow for object cancelation?
Thanks!

**Comment by Manuelrios509** [comment]
I added it, and added exclude objects in slicer and it works pretty good

**Comment by xblax** [comment]
No, this config in Moonraker config is not needed for object cancellation, if a recent slicer is used. It's a legacy option that is no necessary, since Slicers support that out of the box for a long time. Object cancellation is working without issues already.
